### ISSUE TITLE
Include "About Tatoeba" information on the site

### ISSUE THREAD

**Issue by alanfgh** [Tatoeba/tatoeba2#269]
Someone asked where the name "Tatoeba" came from, which made me think that it would be nice to have an "About Tatoeba" page with a little history. Some of the content might be similar to what's on the [Wikipedia page about Tatoeba](http://en.wikipedia.org/wiki/Tatoeba). We do link to the blog, where it's possible to find such information, but only in a circuitous way.

**Comment by trang** [comment]
This information should be added into our ["About" page](https://tatoeba.org/eng/about) which can be accessed from the footer ("What is Tatoeba?"). This page currently only has a video, and definitely needs a bit more text.

I'd like to mark this as "accepted" but before, we would need to come up with a good text to add in our About page.

**Comment by alanfgh** [comment]
What do you think about the text on the Wikipedia page as a starting point? Naturally, we shouldn't just copy and paste, but I'm curious whether you think it does a good job of describing the project.

**Comment by trang** [comment]
There is some inaccurate/wrong information in the Wikipedia article, and it feels too technical. I wouldn't aim to add all the information in the Wikipedia article in the scope of this issue. We should just to start with a small introduction paragraph, in which we can explain why the name "Tatoeba". Other information can be added later on.

We can definitely get inspiration from the article though:

> Tatoeba.org is a free collaborative online database of example sentences geared towards foreign language learners. Its name comes from the Japanese term "tatoeba" (例えば　tatoeba), meaning "for example". Unlike other online dictionaries, which focus on words, Tatoeba focuses on translation of complete sentences.

From the [Tatoeba GSoC ideas page](http://en.wiki.tatoeba.org/articles/show/gsoc_2016_ideas) as well:

> Tatoeba is a platform that aims to build a large database of sentences translated into as many languages as possible. The initial idea was to have a tool in which you could search certain words, and it would return example sentences containing these words with their translations in the desired languages. The name Tatoeba resulted from this concept, because tatoeba means for example in Japanese.

And I'm trying to see if I can access the description that we've used when we submitted our application to GSoC this year, but apparently it's not accessible from the new GSoC website...

**Comment by billoliver** [comment]
My suggestions:

- You can add a list of admins and what they do.
- You can add a list of websites that are using Tatoeba. For example:

**[Wikipedia](https://wikipedia.org)** [**site name**]
A free encyclopedia built collaboratively using wiki software. [**site description**]

**[Many Things](https://manythings.org)** [**app name**]
Interesting Things for ESL/EFL Students. [**app description**]

If you're using Tatoeba on your website or app and you want to add it into this list, [contact us](https://tatoeba.org/eng/contact). (This can be reworded).

**Comment by alanfgh** [comment]
> There is some inaccurate/wrong information in the Wikipedia article, and it feels too technical. I wouldn't aim to add all the information in the Wikipedia article in the scope of this issue. ... We can definitely get inspiration from the article though

Yes, using it for inspiration was what I intended.

I'm not particularly enthusiastic about the idea of adding a list of admins ("The Team") and what they do. It's not as though we have official titles and responsibilities, and I don't think the details of what we do would be of interest to many people who would visit that page.

**Comment by alanfgh** [comment]
> And I'm trying to see if I can access the description that we've used when we submitted our application to GSoC this year, but apparently it's not accessible from the new GSoC website...

There's a short description of Tatoeba visible on the [old website](https://www.google-melange.com/gsoc/org2/google/gsoc2014/tatoeba). I found that link on the Wikipedia page. Here's the text:

> Tatoeba is a libre/free database of example sentences translated into many languages. Our goal is to create a resource for people studying languages—either to learn or research. The database is currently used as a source of example sentences by free dictionaries and language learning websites (like Jim Breen’s WWWJDIC; Jim Breen is actually a member too). It's also used as a rich resource for language learners: They can find out how to use words or how to translate grammatical constructs and idioms. The main site currently has about 1 million page views and 250 thousands unique visitors monthly, as reported by Google Analytics, and the corpus is growing steadily by 3% or more every month.

You probably meant a longer description, though.

**Comment by trang** [comment]
I don't think we need a very long description. 4-5 sentences is enough. As more people ask us questions, we can re-adapt the description. But for a start, I think we should make it short, with just enough information to answer "Where does the name come from?" and a little bit as well "Where does the idea come from?".

This would be by default what I'd use: 

> Tatoeba is a platform that aims to build a large database of sentences translated into as many languages as possible. The initial idea was to have a tool in which you could search certain words, and it would return example sentences containing these words with their translations in the desired languages. The name Tatoeba resulted from this concept, because tatoeba means for example in Japanese.

But I think we can have a better text.

I've got the text we used for GSoC 2016 by the way, and its was a mix of the text above, and of the text from GSoC 2014:

> Tatoeba is a platform that aims to build a large database of example sentences and translations, into as many languages as possible. The initial idea was to have a tool in which you could search certain words, and it would return example sentences containing these words with their translations in the desired languages. The name Tatoeba resulted from this concept, because *tatoeba* means *for example* in Japanese. More than just building a language tool, our goal is also to create an open resource that can be reused by other projects and applications for language learning or linguistic research. Our data is currently used as a source of example sentences by free dictionaries and language learning websites (like [Jim Breen’s WWWJDIC](http://www.edrdg.org/cgibin/
wwwjdic/wwwjdic?1C) or [Lingua.ly](https://lingua.ly/)). A number of [linguistics papers](https://scholar.google.com/scholar?q=tatoeba.org) were also published featuring Tatoeba's data.

**Comment by ckjpn** [comment]
> There is some inaccurate/wrong information in the Wikipedia article .... https://github.com/Tatoeba/tatoeba2/issues/269#issuecomment-199248315

TRANG, why don't you edit the Wikipedia article?  It's a wiki, so you can make corrections.

**Comment by ckjpn** [comment]
## Suggested Rewrite of https://dev.tatoeba.org/eng/about

The Tatoeba Project is a group of volunteers working on a corpus of sentences and their translations that can used by various projects under the Creative Commons - Attribution 2.0 license.

The name of the project comes from the Japanese word <i>tatoeba</i> (例えば) which means "for example."  This was chosen because our website is a collection of example sentences.


## Notes

LINK for "various projects" http://a4esl.com/temporary/tatoeba/links.html
(Eventually, something like this should be part of the project's website, I think.)


"a group of volunteers" could be changed to "a group of people who volunteer their time".

## Why I Suggest Making Changes

> Tatoeba is a platform that aims to build a large database of sentences translated into as many languages as possible. 

The way this is worded talks more about the programming and less about what the project is.

> The initial idea was to have a tool in which you could search certain words, and it would return sentences containing these words with their translations in the desired languages.

Talking about the "initial aim" and "tool" seems to sort of miss the point of the current project and seems to be aimed more at programmers than (potential) contributors.

## Other Ideas

* Include a link to a page with a brief history of the project.
* Include something about the sentence examples can be used to help you study a foreign language.

For example, get a random selection of French sentences with audio linked to English sentences with audio.

https://tatoeba.org/eng/sentences/search?query=&from=fra&to=eng&orphans=no&unapproved=no&user=&tags=&has_audio=yes&trans_filter=limit&trans_to=eng&trans_link=&trans_user=&trans_orphan=&trans_unapproved=&trans_has_audio=yes&sort=random

**Comment by ckjpn** [comment]
## Another Idea
I'm not sure this is a good idea, but it is something to consider.

Include an invitation to join in the right, currently empty 
![screen shot 2016-07-23 at 09 24 36](https://cloud.githubusercontent.com/assets/7934273/17074602/f2df2478-50b7-11e6-82ae-dc43200c9a58.png)
column of the page.

**Comment by alanfgh** [comment]
My comment, from seeing it live on the website:

Replace this:

The initial idea was to have a tool in which you could search certain words, and it would return sentences containing these words with their translations in the desired languages. The name Tatoeba resulted from this concept, because tatoeba means for example in Japanese. 

with this:

Tatoeba gives you a tool to specify words that interest you. It returns sentences containing these words with their translations in the desired languages. The name Tatoeba (_for example_ in Japanese) captures this concept. 

In other words:
- Not only was the word search the initial idea -- it was actually implemented and is the core of Tatoeba.
- You specify words that are of interest to you. (I figure that the word "interest" should catch people's interest, so to speak.)

**Comment by alanfgh** [comment]
Also, when I go to a "About X" page on website X, I expect to find a little bit about the people behind it and whether the site is run for profit. It would be nice to add a sentence like this:

"Tatoeba was founded by Trang Ho in 20xx. Since then, it has received voluntary contributions by thousands of authors of sentences and programmers."

**Comment by ckjpn** [comment]
## Here is another possible sentence to us.

Tatoeba.org is a large database of example sentences translated into many languages by its members who volunteer their time.

**Comment by trang** [comment]
I modified the information text as followed:

> Tatoeba is a large database of sentences and translations. Its content is ever-growing and results from the voluntary contributions of thousands of members.

> Tatoeba provides a tool for you to see examples of how words are used in the context of a sentence. You specify words that interest you, and it returns sentences containing these words with their translations in the desired languages. The name Tatoeba (*for example* in Japanese) captures this concept.

> The project was founded by Trang Ho in 2006, hosted on Sourceforge under the codename of multilangdict.

> The video below, made in 2010, presents the core ideas of Tatoeba. 

I'll close this issue now, but feel free to add additional suggestions of improvements in the comments here.

**Comment by alanfgh** [comment]
Sounds good to me.
